### DISCUSSION TITLE
Use wire:model.defer for input fields in Hub

### DISCUSSION THREAD

**Discussion by gavinhewitt** [lunarphp/lunar#1063] [Feature Requests]
- GetCandy version: latest

Not a bug but fields in hub are defined as `wire:model` instead of `wire:model.defer`, resulting in unnecessary roundtrips to the server. In case of forms where at the end you have a save button, `wire:model.defer` makes more sense.

**Comment by glennjacobs** [comment]
We do use defer in a number of places, but it could be added to others. If you have any specific ones that you think need this adding we'd welcome a PR. 

Also, worth noting I believe Livewire 3 will make defer the default.
